Mollie.Api/Mollie.Api.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.2;net45</TargetFrameworks>
```

Comment:
Hi @Sippelberg ,

I think we need to keep the TargetFramework the way it was, otherwise a lot of other people won't be able to use the library. Is there any particular reason you changed these settings?

Reply:
Hey Vincent, 
You can drop this change, I shouldn't have pushed this update. I needed net48 because other packages we use can only be used with net48.